partcad/src/partcad/shape.py

```
@@ -402,21 +470,21 @@ def render_png(
         filepath=None,
         width=None,
         height=None,
-    ):
+    ) -> None:
         asyncio.run(self.render_png_async(ctx, project, filepath, width, height))
 
     async def render_step_async(
         self,
         ctx,
```

Ruff (0.8.2)

Missing type annotation for function argument `ctx`

(ANN001)